internal/storage/common/errors.go

```diff
@@ -0,0 +1,7 @@
+package common
+
+import "errors"
+
+func ErrNotFound() error {
```

Comment:
1. Идиоматично будет:

```suggestion
var ErrNotFound = errors.New("not found")
```

2. Мне не нравится, что  пакет называется `common`: непонятно для чего он. Лучше назови пакет `errors` и работай с ним